airflow/contrib/executors/kubernetes_executor.py

```diff
@@ -264,6 +264,14 @@ def _validate(self):
                 'must be set for authentication through user credentials; '
                 'or `git_ssh_key_secret_name` must be set for authentication '
                 'through ssh key, but not both')
+        if conf.getboolean(self.kubernetes_section, 'in_cluster') is False \
+           and not conf.get(self.kubernetes_section, 'config_file') \
+           and not conf.get(self.kubernetes_section, 'cluster_context'):
```

Comment:
This change should go now.

Reply:
@ashb Thanks for helping on this PR. Changes are incorporated